Samples/DecoderSample/appsettings.Development.json

```diff
@@ -1,9 +1,9 @@
-{
-  "Logging": {
-    "LogLevel": {
-      "Default": "Debug",
-      "System": "Information",
-      "Microsoft": "Information"
-    }
-  }
-}
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",

```

Comment:
Is it wanted that we change the default value here?

Reply:
Good point. I changed it back to the previous, original values.